pyrokinetics/gs2.py

```diff
@@ -179,6 +179,23 @@ def write(self, pyro, filename, directory='.'):
         # Numerics
         numerics = pyro.numerics
 
+        # Set no. of fields
+        if numerics.phi:
+            gs2_input['knobs']['fphi'] = 1.0
+        else:
+            gs2_input['knobs']['fphi'] = 0.0
+
+        if numerics.apar:
+            gs2_input['knobs']['fapar'] = 1.0
+        else:
+            gs2_input['knobs']['fapar'] = 0.0
+
+        if numerics.bpar:
+            gs2_input['knobs']['fbpar'] = 1.0
+        else:
+            gs2_input['knobs']['fbpar'] = 0.0
```

Comment:
These could use the ternary operator:

```suggestion
        gs2_input['knobs']['fphi'] = 1.0 if numerics.phi else 0.0
        gs2_input['knobs']['fapar'] = 1.0 if numerics.apar else 0.0
        gs2_input['knobs']['fbpar'] = 1.0 if numerics.bpar else 0.0
```

Or even shorter, but perhaps less explicit:

```suggestion
        gs2_input['knobs']['fphi'] = float(numerics.phi)
        gs2_input['knobs']['fapar'] = float(numerics.apar)
        gs2_input['knobs']['fbpar'] = float(numerics.bpar)
```